src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -114,7 +115,30 @@ function repeatableInputToObj(container_name) {
          */
         function bpFieldInitRepeatableElement(element) {
 
-            var field_name = element.attr('name');
+            let field_name = element.attr('name');
+            let form = element.closest('form');
+            let form_events = jQuery._data(form[0], 'events');
+
+            // create a new event handler that will parse the repeatable values to the hidden inputs
+            // so they can be submited along with form when requesting some ajax endpoint
+            // we check that the event is not registered twice
+            if (!('backpack_field' in form_events) || !Object.values(form_events.backpack_field).find(e => e.namespace === 'parse_value')) {
+
+                form.on('backpack_field.parse_value', function(evt, element) {
+                    // TODO: element is the input that made the ajax request
```

Comment:
@pxpm what should we do with the element?
Should it go on the request too?

Reply:
@promatik, select2_from_ajax normally includes the data in the request:
> only send the current field through AJAX

so I believe it should be included.

When
```php
'include_all_form_fields'  => true
```
I would expect that all the fields are sent.

Reply:
@promatik 

Indeed, that is the answer to: #3440 

I just didn't pushed it here because I was planning to create a separate PR for that issue. 

This PR does what it states, "allows the repeatable fields to be sent with ajax requests" and the little extra I left as a `TODO`, that is we already have the "trigger" there.

Reply:
Ok sure, makes sense 😅